pkg/sdk/context.go

```diff
@@ -0,0 +1,45 @@
+package sdk
+
+import (
+	"fmt"
+	"github.com/lunarway/shuttle/pkg/config"
+	"gopkg.in/yaml.v2"
+	"io/ioutil"
+	"path"
+)
+
+type ShuttleContext struct {
+	Variables                 config.DynamicYaml `yaml:"vars"` //temporarily include a dynamic representation of the variables here so the go-based plans can use this for templating so we're backwards compatible with the existing templates (for the time being)
+	ProjectPath               string
+	LocalPlanPath             string
+	LocalShuttleDirectoryPath string
+	TempDirectoryPath         string
```

Comment:
Do we need to annotate these as ignored by the yaml parser? As they are never provided from the `shuttel.yaml` file?

```suggestion
	ProjectPath               string `yaml:"-"`
	LocalPlanPath             string `yaml:"-"`
	LocalShuttleDirectoryPath string `yaml:"-"`
	TempDirectoryPath         string `yaml:"-"`
```